fdbmonitor/fdbmonitor.cpp

```diff
@@ -1171,7 +1183,7 @@ int main(int argc, char** argv) {
 	// testPathOps(); return -1;
 
 	std::string lockfile = "/var/run/fdbmonitor.pid";
-	std::string _confpath = "/etc/foundationdb/foundationdb.conf";
+	std::string _confpath = "/usr/local/etc/foundationdb/foundationdb.conf";
```

Comment:
This probably isn't an acceptable change to do on all platforms.  Is a list of potential config paths needed?  Or use Platform::getDefaultConfigPath?

Reply:
yes, getDefaultConfigPath now has a `defined(__FreeBSD__)` to handle this, so:

```diff
+       std::string _confpath = joinPath(platform::getDefaultConfigPath(), "foundationdb.conf");
```

should be ok.

Reply:
I was unable to find a solution using `platform::getDefaultConfigPath()` without the building failing due to ambiguous definitions. If you can resolve that, we can drop the `#define` garbade.

Reply:
Ambiguous definitions of getDefaultConfigPath sounds weird, because it exists basically for this exact use case.  Let's see if we can hunt down what went wrong.  It sounds plausibly similar to what happened with abspath below.